fdbserver/workloads/RemoveServersSafely.actor.cpp

```diff
@@ -296,13 +296,14 @@ struct RemoveServersSafelyWorkload : TestWorkload {
 		TraceEvent("RemoveAndKill").detail("Step", "exclude list first").detail("ToKill", describe(toKill1)).detail("KillTotal", toKill1.size()).detail("ClusterAvailable", g_simulator.isAvailable());
 		self->excludeAddresses(toKill1);
 
-		Optional<Void> result = wait( timeout( removeAndKill( self, cx, toKill1, NULL), self->kill1Timeout ) );
+		Optional<Void> result = wait( timeout( removeAndKill( self, cx, toKill1, NULL, false), self->kill1Timeout ) );
 
 		bClearedFirst = result.present();
-
+		// killProcArray is always empty here so why are we tracing it? is it meant to be something else or is a step missing somewhere?
```

Comment:
PTAL

Reply:
It was used later to get the the killable processes, at which time, the `killProcArray` is not longer empty.